components/library/gtk3-nocsd/gtk3-nocsd.p5m

```diff
@@ -22,9 +22,6 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
-file path=etc/X11/xinit/xinitrc.d/0310.gtk3-nocsd mode=0555 preserve=true
-
-file path=usr/bin/$(MACH64)/gtk3-nocsd
```

Comment:
Why no 64bit binary?

Reply:
Because it's a script

Reply:
Will test today on my desktop, sorry for the lag.

Reply:
Did you have chance to test it?

Reply:
I tested it and it seems to work fine. Opened devhelp, yelp, gucharmap: all looking good.